src/cljs/webchange/game_changer/steps/select_template/views_list_item.cljs

```diff
@@ -14,8 +14,17 @@
              :class-name "preview-button"}
      "Preview"]))
 
+(defn- start-button
+  [handler props]
```

Comment:
React component should receive only 1 object with props. Merge `handler` as `:on-click` field into `props`